app/views/layouts/_header.html.slim

```diff
@@ -14,4 +14,6 @@ nav.navbar.navbar-default.navbar-fixed-top.topnav[role="navigation"]
             = link_to('Logout', destroy_user_session_path, :method => :delete)
         - else
           li
-            = link_to('Login', new_user_session_path)
\ No newline at end of file
+            = link_to('Login', new_user_session_path)
+      .col-md-12.breadcrumbs
+        = render_breadcrumbs :separator => " / "
```

Comment:
please fix according to @krisberry comment
https://cloud.githubusercontent.com/assets/15616219/11855033/718688da-a450-11e5-81b2-6e74bd8ede1f.jpg
navbar is fixed
try to scroll down and you'll see error - margin won't help